moonshot/src/recipes/recipe_arguments.py

```diff
@@ -1,30 +1,93 @@
+from typing import Any
+
 from pydantic import BaseModel
 
 
 class RecipeArguments(BaseModel):
-    id: str  # id (str): The ID of the recipe.
+    # id (str): The ID of the recipe.
+    id: str
+
+    # name (str): The name of the recipe.
+    name: str
+
+    # description (str): The description of the recipe.
+    description: str
+
+    # tags (list): The list of tags in the recipe.
+    tags: list[str]
+
+    # categories (list): The list of categories in the recipe.
+    categories: list[str]
 
-    name: str  # name (str): The name of the recipe.
+    # datasets (list): The list of datasets used in the recipe.
+    datasets: list[str]
 
-    description: str  # description (str): The description of the recipe.
+    # prompt_templates (list): The list of prompt templates in the recipe.
+    prompt_templates: list[str]
 
-    tags: list[str]  # tags (list): The list of tags in the recipe.
+    # metrics (list): The list of metrics in the recipe.
+    metrics: list[str]
 
-    datasets: list[str]  # datasets (list): The list of datasets used in the recipe.
+    # attack_modules (list): The list of attack modules in the recipe.
+    attack_modules: list[str]
```

Comment:
for optional fields like attack_modules, do we want to have default values for these fields?
i.e. attack_modules: list[str] = []

Reply:
actually you are right. i removed the default value for the grading scale.
i want the recipe creators to have these fields in the recipe instead of setting it to default, else some fields might actually be missing.